style.css

```diff
@@ -130,3 +130,254 @@ i:hover {
   color: #f98500;
   text-decoration: none;
 }
+
+button:hover {
+  background-color: #000;
+  color: #ff6b00;
+  transition: ease-in 1s;
+}
+
+section {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
[OPTIONAL]

- [ ] I recommend using CSS class selectors instead of CSS tags. As the project gets more complicated using class selectors will help make it easier to manipulate & design your project.